packages/client/src/modules/chat/containers/Chat.native.jsx

```diff
@@ -2,57 +2,68 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { View, KeyboardAvoidingView } from 'react-native';
 import { GiftedChat } from 'react-native-gifted-chat';
+import { compose, graphql } from 'react-apollo/index';
+
 import translate from '../../../i18n';
+import MESSAGES_QUERY from '../graphql/MessagesQuery.graphql';
+import ADD_MESSAGE from '../graphql/AddMessage.graphql';
 
 @translate('chat')
 class Chat extends React.Component {
   static propTypes = {
-    t: PropTypes.func
+    t: PropTypes.func,
+    messages: PropTypes.array,
+    addMessage: PropTypes.func
   };
 
+  constructor(props) {
```

Comment:
Don't create a constructor if you not going to use it. It's unnecessary and just pollutes a class.